spring-boot-admin/src/main/java/org/example/teahouse/sba/SbaApplication.java

```diff
@@ -6,12 +6,10 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.context.metrics.buffering.BufferingApplicationStartup;
-import org.springframework.cloud.client.discovery.EnableDiscoveryClient;
 import org.springframework.context.annotation.Import;
 import org.springframework.context.annotation.PropertySource;
 
 @EnableAdminServer
-@EnableDiscoveryClient
```

Comment:
Good catch, do you want to do this (+ fixing the other files with this annotation) in a separate PR?
If so please mention in the commit message that according to the Spring Cloud docs:
>@EnableDiscoveryClient is no longer required. You can put a DiscoveryClient implementation on the classpath to cause the Spring Boot application to register with the service discovery server.

https://docs.spring.io/spring-cloud-commons/reference/4.1/index.html

Reply:
See #18